### DISCUSSION TITLE
FeedCrawler integration

### DISCUSSION THREAD

**Discussion by rix1337** [sct/overseerr#258] [Feature Requests]
[FeedCrawler](https://github.com/rix1337/FeedCrawler) has a small but devoted following and a few sponsors.
This is in short a web scraper focused on a German audience (for which Radarr/Sonarr do not work well).

Ombi never really worked well for me, and their devs are not interested in integrating FeedCrawler.
As a workaround I have implemented Ombi into my project by calling its API instead:
https://github.com/rix1337/FeedCrawler/blob/master/feedcrawler/ombi.py#L105

I would much rather use this with direct integration into FeedCrawler.

Could you please provide me with instructions what functions/API I need to provide?

I have implemented and can make available:
* Download Movie (IMDb ID)
* Download Show (IMDb ID)
* Download Season (IMDb ID + SXX)
* Download Episode (IMDb ID + SXXEYY)

Additional Option for Movies:
* Add to watchlist if not found

Additional Option for Shows:
* Add to watchlist for upcoming Seasons/Episodes

**Comment by 9Mad-Max5** [comment]
The new name is now FeedCrawler.
But the links still work.

**Reply by rix1337** [reply]
I changed the post accordingly.

**Comment by rix1337** [comment]


**Reply by jankete** [reply]
Done
